polylogue/cli/prefs.py

```
@@ -0,0 +1,77 @@
+from ..paths import STATE_HOME
+from ..commands import CommandEnv
+
+PREFS_PATH = STATE_HOME / "prefs.json"
+
+
+def _load_prefs() -> Dict[str, Dict[str, str]]:
+    if not PREFS_PATH.exists():
+        return {}
+    try:
+        return json.loads(PREFS_PATH.read_text(encoding="utf-8"))
+    except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)